Utilities/Distribution/manylinux/imagefiles/cmd.sh

```diff
@@ -5,6 +5,7 @@ set -x
 export SRC_DIR="/tmp/SimpleITK"
 export BLD_DIR="/tmp/SimpleITK-build"
 export OUT_DIR="/work/io"
+ExternalData_OBJECT_STORES=/home/blowekamp/.ExternalData
```

Comment:
Should this be $HOME/.ExternalData ?

Reply:
Nice catch, that line should not be there.

Reply:
Just a suggestion:

```bash
# Use ExternalData_OBJECT_STORES as a local cache for downloaded objects.
if [ -d ${HOME}/.ExternalData ] & [  -z "${ExternalData_OBJECT_STORES}" ]; then
  ExternalData_OBJECT_STORES=${HOME}/.ExternalData
fi

```

Reply:
Thanks for the comment.

The line below was what I wanted to do:
```
export ExternalData_OBJECT_STORES=${ExternalData_OBJECT_STORES:-/tmp/.ExternalData}
```

This is run in a docker image, frequently driven by [this script](https://github.com/SimpleITK/SimpleITK/blob/cf5c60a4294c8409888cf03d77ab6114a6de45a2/Utilities/Distribution/manylinux/run.sh#L3), which propagates the user env variable, and mount point. When not set, the line below should set up the cache env.

I'll add a comment to clarify.